Makefile

```diff
@@ -48,21 +48,40 @@ NEWEST_GO_FILE = $(shell find $(SRC_DIRS) -name \*.go -exec $(STAT) {} \; \
 TYPES_FILES    = $(shell find pkg/apis -name types.go)
 GO_VERSION     = 1.8
 
+ALL_ARCH=amd64 arm arm64 ppc64le s390x
+
 PLATFORM?=linux
 ARCH?=amd64
 
+# TODO: Consider using busybox instead of debian
+ifeq ($(ARCH),amd64)
+	BASEIMAGE?=debian:jessie
+endif
+ifeq ($(ARCH),arm)
+	BASEIMAGE?=arm32v7/debian:jessie
+endif
+ifeq ($(ARCH),arm64)
+	BASEIMAGE?=arm64v8/debian:jessie
+endif
+ifeq ($(ARCH),ppc64le)
+	BASEIMAGE?=ppc64le/debian:jessie
+endif
+ifeq ($(ARCH),s390x)
+	BASEIMAGE?=s390x/debian:jessie
+endif
+
 GO_BUILD       = env GOOS=$(PLATFORM) GOARCH=$(ARCH) go build -i $(GOFLAGS) \
                    -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION)"
 BASE_PATH      = $(ROOT:/src/github.com/kubernetes-incubator/service-catalog/=)
 export GOPATH  = $(BASE_PATH):$(ROOT)/vendor
 
 MUTABLE_TAG                      ?= canary
-APISERVER_IMAGE                   = $(REGISTRY)apiserver:$(VERSION)
-APISERVER_MUTABLE_IMAGE           = $(REGISTRY)apiserver:$(MUTABLE_TAG)
-CONTROLLER_MANAGER_IMAGE          = $(REGISTRY)controller-manager:$(VERSION)
-CONTROLLER_MANAGER_MUTABLE_IMAGE  = $(REGISTRY)controller-manager:$(MUTABLE_TAG)
-USER_BROKER_IMAGE                 = $(REGISTRY)user-broker:$(VERSION)
-USER_BROKER_MUTABLE_IMAGE         = $(REGISTRY)user-broker:$(MUTABLE_TAG)
+APISERVER_IMAGE                   = $(REGISTRY)apiserver-$(ARCH):$(VERSION)
+APISERVER_MUTABLE_IMAGE           = $(REGISTRY)apiserver-$(ARCH):$(MUTABLE_TAG)
+CONTROLLER_MANAGER_IMAGE          = $(REGISTRY)controller-manager-$(ARCH):$(VERSION)
+CONTROLLER_MANAGER_MUTABLE_IMAGE  = $(REGISTRY)controller-manager-$(ARCH):$(MUTABLE_TAG)
+USER_BROKER_IMAGE                 = $(REGISTRY)user-broker-$(ARCH):$(VERSION)
+USER_BROKER_MUTABLE_IMAGE         = $(REGISTRY)user-broker-$(ARCH):$(MUTABLE_TAG)
```

Comment:
If all of these have the arch name tacked on the end, they'll be differently named from the images in the chart.
For example, https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/catalog/values.yaml#L4  won't match anymore.